composer.json

```diff
@@ -169,17 +181,17 @@
                 "[web-root]/web.config": {
                     "mode": "skip"
                 }
-            },
-            "initial": {
-                "sites/default/default.services.yml": "[web-root]/sites/default/services.yml",
-                "sites/default/default.settings.php": "[web-root]/sites/default/settings.php"
             }
         }
     },
     "scripts": {
         "post-install-cmd": [
             "@prepare"
         ],
+        "post-drupal-scaffold-cmd": [
+            "cp tests/fixtures/project_01/docroot/sites/default/default.settings.php tests/fixtures/project_01/docroot/sites/default/settings.php",
+            "cp tests/fixtures/project_01/docroot/sites/default/default.services.yml tests/fixtures/project_01/docroot/sites/default/services.yml"
+        ],
         "lint": [
             "@lint:phpcs"
```

Comment:
Please run `composer run lint` and fix the warnings/errors.

Reply:
Resolved in [2eab336](https://github.com/Cheppers/drupal-behat-contexts/pull/34/commits/2eab336a83f6e1432565af7dc9f658aef84d322e).